data/secscan_model/secscan_v4_model.py

```diff
@@ -211,10 +212,25 @@ def needs_reindexing_query(indexer_hash):
             ),
         )
 
+        def manifest_unsupported_status(manifest):
+            with db_transaction():
+                ManifestSecurityStatus.delete().where(
+                    ManifestSecurityStatus.manifest == manifest._db_id,
+                    ManifestSecurityStatus.repository == manifest.repository._db_id,
+                ).execute()
+                ManifestSecurityStatus.create(
+                    manifest=manifest._db_id,
+                    repository=manifest.repository._db_id,
+                    index_status=IndexStatus.MANIFEST_UNSUPPORTED,
+                    indexer_hash="none",
+                    indexer_version=IndexerVersion.V4,
+                    metadata_json={},
+                )
+
         for candidate, abt, num_remaining in iterator:
             manifest = ManifestDataType.for_manifest(candidate, None)
             layers = registry_model.list_manifest_layers(manifest, self.storage, True)
-            if layers is None:
+            if layers is None or layers == []:
```

Comment:
helm charts have an empty array of layers, which is not scannable

Reply:
Maybe `len(layers) == 0` might be more readable.